d2/src/main/java/com/linkedin/d2/balancer/simple/SimpleLoadBalancer.java

```diff
@@ -789,24 +805,16 @@ public void onSuccess(ServiceProperties result)
     getLoadBalancedServiceProperties(serviceName, waitForUpdatedValue, callback);
   }
 
-  public void getLoadBalancedServiceProperties(String serviceName, boolean waitForUpdatedValue, Callback<ServiceProperties> servicePropertiesCallback)
+  public void getLoadBalancedServiceProperties(String serviceName, boolean waitForUpdatedValue,
+                                               Callback<ServiceProperties> servicePropertiesCallback)
```

Comment:
Indenting code like this rarely survives refactors. It tends to become more annoying to maintain than anything (for example, if `getLoadBalancedServiceProperties` is ever renamed, nothing will be aligned anymore). I would recommend turning this setting off in your editor, and simply align with the continuation indent:
```suggestion
  public void getLoadBalancedServiceProperties(
      String serviceName,
      boolean waitForUpdatedValue,
      Callback<ServiceProperties> servicePropertiesCallback)
```

Reply:
Thanks for ur suggestion. Done